sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -743,7 +743,7 @@ export class EventHubReceiver extends LinkEntity {
             onSessionError: (context: EventContext) => this._onAmqpSessionError(context)
           };
           if (this.checkpoint > -1) {
-            receiverOptions.eventPosition = EventPosition.fromSequenceNumber(this.checkpoint);
+            receiverOptions.eventPosition ={ sequenceNumber: this.checkpoint};
```

Comment:
Super nit (prettier should fix this):
```suggestion
            receiverOptions.eventPosition = { sequenceNumber: this.checkpoint };
```